core/src/main/java/org/sonarsource/sonarlint/core/container/standalone/StandaloneGlobalContainer.java

```diff
@@ -138,6 +141,7 @@ public AnalysisResults analyze(StandaloneAnalysisConfiguration configuration, Is
     analysisContainer.add(issueListener);
     analysisContainer.add(rules);
     analysisContainer.add(activeRules);
+    activeRules.setExclusions(configuration.excludedRules());
```

Comment:
Is it not dangerous in case of concurrent analysis to have a mutable ActiveRulesWithExclusions? Why not simply replace the previous line by: `analysisContainer.add(new ActiveRulesWithExclusions(container.getActiveRules(), configuration.excludedRules()))`; and revert the change at line 134